bin/ream/src/cli/account_manager.rs

```diff
@@ -14,23 +17,35 @@ pub struct AccountManagerConfig {
 
     /// Account lifetime in 2 ** lifetime slots
     #[arg(short, long, default_value_t = 18)]
-    pub lifetime: u64,
+    pub lifetime: u32,
 
     /// Chunk size for messages
     #[arg(short, long, default_value_t = 5)]
-    pub chunk_size: u64,
+    pub chunk_size: u32,
 
     /// Seed phrase for key generation
-    #[arg(short, long)]
+    #[arg(short, long, conflicts_with = "import_keystore")]
     pub seed_phrase: Option<String>,
 
+    /// Optional BIP39 passphrase used with the seed phrase
+    #[arg(long)]
+    pub passphrase: Option<String>,
+
     /// Activation epoch for the validator
     #[arg(long, default_value_t = DEFAULT_ACTIVATION_EPOCH)]
-    pub activation_epoch: usize,
+    pub activation_epoch: u32,
 
     /// Number of active epochs
     #[arg(long, default_value_t = DEFAULT_NUM_ACTIVE_EPOCHS)]
-    pub num_active_epochs: usize,
+    pub num_active_epochs: u32,
+
+    /// Path for keystore directory
+    #[arg(long, default_value = DEFAULT_KEYSTORE_PATH)]
+    pub path: String,
```

Comment:
shouldn't we be using Ream's data-dir directory not a hidden local folder to the binary

Reply:
The keystore should be stored in its own directory for separation of concerns, rather than being bundled into Ream’s datadir.

Reply:
I still don't see why the default directory should be a hidden path in the current active directory

Reply:
Changed the keystore to be a sub directory of ream's data directory.